manifest-build-tools/application/update_manifest.py

```diff
@@ -0,0 +1,442 @@
+#!/usr/bin/env python
+
+# Copyright 2015, EMC, Inc.
+
+"""
+Usage:
+./on-tools/manifest-build-tools/HWIMO-BUILD on-tools/manifest-build-tools/application/update_manifest.py \
+--repo "$url" \
+--branch "$branch" \
+--commit "$id" \
+--manifest_download_url https://dl.bintray.com/rackhd-mirror/binary/manifest/ \
+--git-credential https://github.com,GITHUB_CREDS \
```

Comment:
It'll clone the repo of the ```--repo "$url"  --branch "$branch" -commit "$id" ```
to check if the parameters are valid.
So, though rackhd is a public repo, the ```--git-credential``` is still needed.